### ISSUE TITLE
Update pyproject metadata

### ISSUE THREAD

**Issue by jernst** [fediverse-devnet/feditest#208]
For better left column in https://pypi.org/project/feditest/0.2/

**Comment by jernst** [comment]
Also, the main page is simply taken from the repo, and does not really meet the needs of the pypi listing.

**Comment by jernst** [comment]
Also, `pip show feditest` has lots of missing fields.

```
Name: feditest
Version: 0.2
Summary: Testing federated protocols
Home-page: 
Author: 
Author-email: Johannes Ernst <git@j12t.org>, Steve Bate <svc-github@stevebate.net>
License: 
Location: /opt/homebrew/lib/python3.11/site-packages
Requires: httpx, jinja2, langcodes, msgspec, multidict, pyhamcrest, pytest
Required-by: 
```

**Comment by jernst** [comment]
Fixed in #229
